### ISSUE TITLE
Java NoClassDefFoundError

### ISSUE THREAD

**Issue by njhanne** [trackmate-sc/TrackMate-Cellpose#4]
Hello!

I am having an issue with the Trackmate cellpose integration. I've tried running it on WIndows and Ubuntu with GPU, as well as Macintosh without GPU. I get the same error every time. Cellpose itself works fine through conda, just not the trackmate plugin.

I tried the LoG detector and that one seems to be fine, so I think it is specific to cellpose?

Thanks for any advice! Can't wait to see if this tool will work for this project!

```
Using built-in defaults.
Exception in thread "Wizard exec thread" java.lang.NoClassDefFoundError: fiji/plugin/trackmate/detection/LabeImageDetectorFactory
	at fiji.plugin.trackmate.cellpose.CellposeDetector.process(CellposeDetector.java:295)
	at fiji.plugin.trackmate.TrackMate.processGlobal(TrackMate.java:402)
	at fiji.plugin.trackmate.TrackMate.execDetection(TrackMate.java:371)
	at fiji.plugin.trackmate.gui.wizard.descriptors.ExecuteDetectionDescriptor.lambda$getForwardRunnable$0(ExecuteDetectionDescriptor.java:55)
	at fiji.plugin.trackmate.gui.wizard.WizardController$1.run(WizardController.java:191)
Caused by: java.lang.ClassNotFoundException: fiji.plugin.trackmate.detection.LabeImageDetectorFactory
	at java.net.URLClassLoader.findClass(URLClassLoader.java:387)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
	... 5 more
```

![Screenshot from 2023-01-20 15-11-27](https://user-images.githubusercontent.com/5288493/213821425-b6018f12-310a-48a5-bb60-fabfd4186a54.png)

![Screenshot from 2023-01-20 15-13-06](https://user-images.githubusercontent.com/5288493/213821552-000bbf88-fe41-4061-ac17-e5521ce8419f.png)

**Comment by tinevez** [comment]
Well done!
This is because of the update of TrackMate in v7.10.0
I fixed the name of the `LabeLimageDetector` but forgot to correct it in other modules.
I will do this.

**Comment by tinevez** [comment]
Fixed in dc9052a3992b70b17a9238ea763ee4e982180760

**Comment by njhanne** [comment]
Merci! I will give it another try today!
